renovate.json

```diff
@@ -11,6 +11,10 @@
     {
       "matchPackageNames": ["langchain-openai"],
       "allowedVersions": "<=0.1.7"
+    },
+    {
+      "matchPackageNames": ["pymilvus"],
+      "allowedVersions": "<=2.4.2"
```

Comment:
shouldnt this be `<=2.4.1` or `<2.4.2` as the [dep version](https://github.com/containers/ai-lab-recipes/blob/e46a5105ed3ad71b102f94b89acbdf95e8ee75ec/recipes/natural_language_processing/rag/app/requirements.txt#L7) is `2.4.1`?

Reply:
@Gregory-Pereira fixed